aws_database/db_service/models.py

```diff
@@ -0,0 +1,177 @@
+from sqlalchemy import Column, ForeignKey, Integer, String, Boolean, Numeric, DateTime, Text, Date, JSON
+from sqlalchemy.orm import relationship
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.dialects.postgresql import DOUBLE_PRECISION, TIMESTAMP, JSONB
+
+Base = declarative_base()
+
+class AssessmentResult(Base):
+    __tablename__ = 'assessmentResult'
+    
+    id = Column(Integer, primary_key=True, index=True)
+    assessment_id = Column(Integer, ForeignKey('assessment.id'), nullable=False)
+    score = Column(Numeric)
+    flag = Column(Boolean)
+    note = Column(Text)
+    vref = Column(Text, ForeignKey('verseReference.fullverseid'))
```

Comment:
vref is <BOOK>:<CHAPTER>: <VERSE>

I currently have a postgres operation which I think automatically takes vref and auto-populates Book, Chapter and Verse columns in the table, for faster filtering of queries. But I'm wondering if it makes sense to add those Book, Chapter and Verse fields here, since we have the opportunity.

What do you think?

Reply:
Oh sorry - we already have that. Ignore my previous comment! :smile:

Reply:
But a note to self that with this schema we'll be able to manually push the book, chapter and verse fields to the database, instead of the server-side postgres operation we currently have.